templates/next/components/foo/Form.tsx

```diff
@@ -59,6 +59,13 @@ export const Form: FunctionComponent<Props> = ({ {{{lc}}} }) => {
               isValid: false,
               msg: `Error when ${isCreation ? 'creating': 'updating'} the resource.`,
             });
+             const apiErrorsMsg = Object.keys(error.errors)
```

Comment:
This should be done directly in `dataAccess.ts`, so it will not be copy/pasted in every form component.